sand/controllers/search.py

```diff
@@ -33,23 +36,26 @@ def search_classes():
     """API Route to search for classes with their names"""
     search_text = request.args.get('q')
     wikidata_search = get_search('classes')
-    payload = wikidata_search.find_class_by_name(search_text)
-    return jsonify(payload)
+    search_payload = wikidata_search.find_class_by_name(search_text)
+    serialized_payload = serializer(search_payload)
```

Comment:
serializer won't work on list of objects

Reply:
fixed this by added a serializer for searchItem